src/utils/auth.js

```diff
@@ -0,0 +1,19 @@
+import jwtDecode from 'jwt-decode';
+
+export const isLoggedIn = () => {
+  const { currentUser } = JSON.parse(localStorage.getItem('persist:hackton'));
+  const { token } = JSON.parse(currentUser);
+  if (token) {
+    const decodedToken = jwtDecode(token);
+
+    const currentTime = Date.now() / 1000;
+
+    if (decodedToken.exp < currentTime) {
+      // Logout user
+      return false;
```

Comment:
Can we create a `useAuth` hook for this, to avoid trying to fetch the information from localStorage?

Reply:
ok. But I think we can roll this out for our RC1 approval and refactor afterwards